build-config/checkstyle/build/checkstyle.xml

```diff
@@ -1,407 +0,0 @@
-        MINUS_ASSIGN, MOD, MOD_ASSIGN, NOT_EQUAL, PLUS, PLUS_ASSIGN, QUESTION,
-        SL, SLIST, SL_ASSIGN, SR_ASSIGN, STAR, STAR_ASSIGN" />
-            <property name="allowEmptyConstructors" value="true" />
-            <property name="allowEmptyMethods" value="true" />
-            <property name="severity" value="error" />
-        </module>
-
-        <module name="WhitespaceAfter">
-            <!-- Checks that commas, semicolons and typecasts are followed by
-                 whitespace.
-            -->
-            <property name="tokens" value="COMMA, SEMI, TYPECAST" />
-            <property name="severity" value="error" />
-        </module>
-
-        <module name="NoWhitespaceAfter">
-            <!-- Checks that there is no whitespace after various unary operators.
-                 Linebreaks are allowed.
-            -->
-            <property name="tokens" value="BNOT, DEC, DOT, INC, LNOT, UNARY_MINUS,
-        UNARY_PLUS" />
-            <property name="allowLineBreaks" value="true" />
-            <property name="severity" value="error" />
-        </module>
-
-        <module name="NoWhitespaceBefore">
-            <!-- Checks that there is no whitespace before various unary operators.
-                 Linebreaks are allowed.
-            -->
-            <property name="tokens" value="SEMI, DOT, POST_DEC, POST_INC" />
-            <property name="allowLineBreaks" value="true" />
-            <property name="severity" value="error" />
-        </module>
-
-        <module name="ParenPad">
-            <!-- Checks that there is no whitespace before close parens or after
-                 open parens.
-            -->
-            <property name="severity" value="error" />
-        </module>
-
-        <!-- No System.out -->
-        <module name="Regexp">
-            <property name="format" value="System\.out\.println" />
-            <property name="illegalPattern" value="true" />
-        </module>
-
-        <!-- No System.err -->
-        <module name="Regexp">
-            <!-- . matches any character, so we need to escape it and use \. to match dots. -->
-            <property name="format" value="System\.err\.println" />
-            <property name="illegalPattern" value="true" />
-        </module>
-
-        <!-- No printStackTrace -->
-        <module name="Regexp">
-            <!-- . matches any character, so we need to escape it and use \. to match dots. -->
-            <property name="format" value="e\.printStackTrace\(\)" />
-            <property name="illegalPattern" value="true" />
-        </module>
-
-        <module name="SuppressionCommentFilter">
-            <property name="offCommentFormat" value="CHECKSTYLE OFF: (.+)"/>
-            <property name="onCommentFormat" value="CHECKSTYLE ON"/>
-            <property name="checkFormat" value="Javadoc.*"/>
-            <property name="messageFormat" value="$1"/>
-        </module>
-
-    </module>
-
-</module>
```

Comment:
Any reason to delete this file?

Reply:
We have a Gradle task to download the checkstyle XML files. So we don't need the hard coded files committed.